tests/test_coef.py

```diff
@@ -290,6 +290,7 @@ def test_cm_basic():
     cm_value = ccc(feature0, feature1)
     assert cm_value is not None
     assert isinstance(cm_value, float)
+    assert 0.00 < cm_value < 0.02
```

Comment:
If the `cm_value` is supposed to be .01, you can could you `assert cm_value == pytest.approx(0.01)` here to be more explicit

Reply:
Oh my God, this is great! Thank you!